### PR TITLE
chore: use eslint-config-kodefox

### LINKED ISSUE #385: add eslint-config-kodefox on miflex and flexship
We should also install `eslint-config-kodefox` on remaining projects:
- [x] miflex
- [x] flexship

### PR DESCRIPTION
**Pull request by oshimayoan**
Resolves #385 and #386.

### CODE REVIEW

File: packages/feetch/package.json [hunk 1]
```diff
@@ -34,12 +34,8 @@
     "@testing-library/react-hooks": "3.2.1",
     "@types/jest": "25.1.4",
     "@types/react": "16.9.23",
-    "@typescript-eslint/eslint-plugin": "2.23.0",
-    "@typescript-eslint/parser": "2.23.0",
     "eslint": "6.8.0",
     "eslint-config-kodefox": "0.1.0",
```
Comment (danielsukmana) [inline packages/feetch/package.json]: should we update this to the latest?

File: packages/feetch/package.json [hunk 2]
```diff
@@ -34,12 +34,8 @@
...
     "@types/jest": "25.1.4",
     "@types/react": "16.9.23",
-    "@typescript-eslint/eslint-plugin": "2.23.0",
-    "@typescript-eslint/parser": "2.23.0",
     "eslint": "6.8.0",
-    "eslint-config-kodefox": "0.1.0",
-    "eslint-config-prettier": "6.10.0",
-    "eslint-plugin-prettier": "3.1.2",
+    "eslint-config-kodefox": "^0.2.2",
     "eslint-plugin-react": "7.19.0",
```
Comment (darcien) [inline packages/feetch/package.json]: I think we can also remove this one

Comment (oshimayoan) [reply packages/feetch/package.json]: Does it because `eslint-config-kodefox` has it as `optionalDep`?

Comment (darcien) [reply packages/feetch/package.json]: Correct!

**Pull request merged**